code/datums/migrants/migrant_waves/knightly_journey_roles.dm

```diff
@@ -0,0 +1,131 @@
+	belt = /obj/item/storage/belt/rogue/leather/steel/tasset
+	backl = /obj/item/storage/backpack/rogue/satchel
+	beltl = /obj/item/flashlight/flare/torch/lantern
+	backpack_contents = list(
+		/obj/item/storage/belt/rogue/pouch/coins/poor = 1,
+		/obj/item/recipe_book/survival = 1,
+		)
+	H.adjust_skillrank(/datum/skill/combat/polearms, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/swords, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/maces, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/axes, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/misc/riding, 4, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/crossbows, 4, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/bows, 4, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/wrestling, 4, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/unarmed, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/misc/climbing, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/misc/reading, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/misc/athletics, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/knives, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/misc/tracking, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/misc/swimming, 2, TRUE)
+	H.dna.species.soundpack_m = new /datum/voicepack/male/knight()
+	H.set_blindness(0)
+	var/weapons = list("Longsword","Mace + Shield","Flail + Shield","Billhook","Battle Axe","Greataxe")
+	var/weapon_choice = input(H, "Choose your weapon.", "TAKE UP ARMS") as anything in weapons
+	switch(weapon_choice)
+		if("Longsword")
+			H.adjust_skillrank_up_to(/datum/skill/combat/swords, SKILL_LEVEL_EXPERT, TRUE)
+			beltr = /obj/item/rogueweapon/sword/long
+			r_hand = /obj/item/rogueweapon/scabbard/sword
+		if("Mace + Shield")
+			H.adjust_skillrank_up_to(/datum/skill/combat/maces, SKILL_LEVEL_EXPERT, TRUE)
+			H.adjust_skillrank_up_to(/datum/skill/combat/shields, SKILL_LEVEL_MASTER, TRUE)
+			beltr = /obj/item/rogueweapon/mace
+			backr = /obj/item/rogueweapon/shield/tower/metal
+		if("Flail + Shield")
+			H.adjust_skillrank_up_to(/datum/skill/combat/whipsflails, SKILL_LEVEL_EXPERT, TRUE)
+			H.adjust_skillrank_up_to(/datum/skill/combat/shields, SKILL_LEVEL_MASTER, TRUE)
+			beltr = /obj/item/rogueweapon/flail
+			backr = /obj/item/rogueweapon/shield/tower/metal
+		if("Billhook")
+			H.adjust_skillrank_up_to(/datum/skill/combat/polearms, SKILL_LEVEL_EXPERT, TRUE)
+			r_hand = /obj/item/rogueweapon/spear/billhook
+			backr = /obj/item/rogueweapon/scabbard/gwstrap
+		if("Battle Axe")
+			H.adjust_skillrank_up_to(/datum/skill/combat/axes, SKILL_LEVEL_EXPERT, TRUE)
+			r_hand = /obj/item/rogueweapon/stoneaxe/battle
+		if("Greataxe")
+			H.adjust_skillrank_up_to(/datum/skill/combat/axes, SKILL_LEVEL_EXPERT, TRUE)
+			r_hand = /obj/item/rogueweapon/greataxe
+			backr = /obj/item/rogueweapon/scabbard/gwstrap
+
+/datum/migrant_role/kj_squire
+	name = "Squire"
+	advclass_cat_rolls = list(CTAG_KG_SQUIRE = 20)
+
+/datum/advclass/kj_squire
+	name = "Squire"
+	outfit = /datum/outfit/job/roguetown/adventurer/squire
```

Comment:
No unique outfit? Probably for the best, but still figure I should point this out.

Reply:
Nadda. The adventurer one works as is. Just figured I'd point it out, in case it wasn't intended.